.github/actions/molecule_integration_ec2/action.yml

```diff
@@ -64,13 +61,19 @@ runs:
         echo "NEXUS_USERNAME=${{ inputs.nexus_username }}" >> $GITHUB_ENV
         echo "NEXUS_PASSWORD=${{ inputs.nexus_password }}" >> $GITHUB_ENV
 
+    - name: Login to AWS
+      uses: aws-actions/configure-aws-credentials@e3dd6a429d7300a6a4c196c26e071d42e0343502 # v4.0.2
+      with:
+        aws-region: ${{ inputs.aws_region }}
+        role-to-assume: arn:aws:iam::372466110691:role/AlfrescoCI/alfresco-cloud-deploy
+        role-session-name: ${{ github.event.repository.name }}-${{ github.run_id }}
+        role-duration-seconds: 5400
```

Comment:
we should probably document in the oidc page on confluence that this is probably the best approach to build a session name and what is needed when the default duration is not enough (for longer than usual workflows)